docs/installation.rst

```diff
@@ -6,60 +6,80 @@ General
 
 This is a python 3.8 project.
 
-In general, you can install the scripts from Pypi by run ``python -m pip install ifiscripts`` on Windows or ``python3 -m pip install ifiscripts`` on Mac OS.
+In general, you can install the scripts from Pypi (installed along with Python) by running ``python -m pip install ifiscripts`` on Windows or ``python3 -m pip install ifiscripts`` on Mac OS.
 
-You can also clone or download the whole repository (https://github.com/Irish-Film-Institute/IFIscripts) and run the scripts from your cloned path. In the Irish Film Institute, on linux, Mac OS and Windows, we create a folder in the home directory called ``ifigit``, then we run ``git clone https://github.com/irish-film-institute/ifiscripts``. Then we add the ``ifiscripts`` folder to ``$PATH`` which allows us to access the scripts from any directory, not just ``ifigit/ifiscripts``.
+You can also clone or download the whole repository (https://github.com/Irish-Film-Institute/IFIscripts) and run the scripts from your cloned path. 
 
-However some folks just ``cd`` into the cloned repository and run the scripts from there, for example to run ``makeffv1.py`` you might run:
-``python makeffv1.py path/to.filename.mov``.
+For our developer, on Mac OS and Windows, we create a folder in the home directory called ``ifigit``, then we run ``git clone https://github.com/Irish-Film-Institute/IFIscripts.git``. Then we add the ``ifiscripts`` folder to ``$PATH`` which allows us to access the scripts from any directory, not just from ``ifigit/ifiscripts``.
```

Comment:
'For our development workflows' ?

Reply:
Agreed. There's no one else and developer sounds lonely...

Reply:
This will be put into a new PR later.